graph.cpp

```diff
@@ -27,13 +27,29 @@ Graph::Graph(uint64_t num_nodes): num_nodes(num_nodes) {
   GraphWorker::startWorkers(this, bf);
 }
 
+Graph::Graph(std::ifstream& file) {
```

Comment:
So the way we're thinking about using this in multiple CC is to reset an existing graph's sketches using the data on disk. How would we accomplish that using this setup?

Reply:
Constructing from a filestream saves only the supernode and metadata fields. It wasn't made with the intention of being able to be a functional ingestion machine after construction. That's why I use the term "reheated" like you would yesterday's dinner. I'll rethink this whole process in light of our new need for a snapshot-based serialization.